x-pack/plugins/actions/server/plugin.test.ts

```diff
@@ -72,7 +72,9 @@ describe('Actions Plugin', () => {
     });
 
     it('should log warning when Encrypted Saved Objects plugin is using an ephemeral encryption key', async () => {
-      await plugin.setup(coreSetup, pluginsSetup);
+      // coreMock.createSetup doesn't support Plugin generics
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      await plugin.setup(coreSetup as any, pluginsSetup);
```

Comment:
I'll open an issue with Platform - sadly this piece of generics typing was missed in our core mocks